azure_arc_vsphere_jumpstart/resource_bridge/powershell/vCenter_onboarding.ps1

```diff
@@ -0,0 +1,250 @@
+# <--- Change the following environment variables according to your environment --->
+
+$location = '<Azure region>'
+$SubscriptionId = '<Subscription ID>'
+$ResourceGroupName = '<Azure Resource Group Name>'
+$applianceName = '<bridge appliance name>'
+$customLocationName = '<Custom Location name>'
+$vCenterName = '<vCenter name>'
+$vcenterfqdn = '<vCenter FQDN>'
+$vcenterusername = '<vCenter Username>'
+$vcenterpassword = '<vCenter Password>'
+$appID = '<Service principal AppID>'
+$password = '<Service principal password>'
+$tenantId = '<Tenant ID>'
+$vSphereRP = '<ConnectedVMwarevSphere resource provider Id>'
+
```

Comment:
Be consistent with the parameters file convention. you have a mix of just lowercase and lowercase/uppercase